Makefile

```diff
@@ -372,6 +375,27 @@ verify-gen: generate ## Verify generated files
 vet: ## Runs the Go vet command on this project
 	go vet ./...
 
+## --------------------------------------
+## Tilt / Kind
+## --------------------------------------
+
+.PHONY: kind-create
+kind-create: ## create capdo kind cluster if needed
+	./scripts/kind-with-registry.sh
+
+.PHONY: tilt-up
+tilt-up: kind-create ## start tilt and build kind cluster if needed
+	tilt up
```

Comment:
actually we will run tilt up in the `cluster-api` directory instead of this directory.
why we need this?

Reply:
hum good catch, thanks!